### ISSUE TITLE
Bruno crashes when viewing `Timeline` for a POST request with a multipart form

### ISSUE THREAD

**Issue by cb80** [usebruno/bruno#2997]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

Bruno crashes when viewing the `Timeline` for a POST request with a multipart form.

Steps to reproduce:
- create a request based on the given `.bru` file
- run the request
- click the `Timeline` button

Bruno crashes with the `Oops! Something went wrong` message.

The issue does not occur if the body is e.g. json. So, it seems to be related to `multipart-form`.

### .bru file to reproduce the bug

```bru
meta {
  name: test
  type: http
  seq: 2
}

post {
  url: https://echo.zuplo.io
  body: multipartForm
  auth: none
}

body:multipart-form {
  foo: bar
}
```

### Screenshots/Live demo link

<img width="922" alt="image" src="https://github.com/user-attachments/assets/052cc59c-bf99-4664-a644-c41cb7a75d20">

**Comment by helloanoop** [comment]
Thanks @cb80 for bringing this to our notice. We were able to reproduce this issue on our end.
We have a release outgoing tomorrow and will have a fix for this issue included in tomorrow's release.

**Comment by helloanoop** [comment]
Just merged the PR by @lohxt1 !
This will be available in the upcoming `v1.28.0` release.
